### PR TITLE
Clickcode is stupid

### LINKED ISSUE #22031: Nonhumans can't ctrl+click to drag humans
I was an engiborg with no modules active. Ctrl+click on laying mobs did not drag them. Ctrl+click on items dragged them as normal.

I could still right click+pull mobs, and ctrl+clicking floors let go of them as normal.

Duncdar confirmed as well as a janiborg

### PR DESCRIPTION
**Pull request by ExcessiveUseOfCobblestone**
:cl: Cobby
fix: You can pull as other mobs now. Sorry, clickcode is stupid.
/:cl:

fixes #22031

### CODE REVIEW

**Comment by optimumtact** [comment]
COBBY

File: code/_onclick/click.dm [hunk 1]
```diff
@@ -234,6 +234,7 @@
 		var/mob/living/carbon/human/H = user
 		H.dna.species.grab(H, src, H.martial_art)
 		H.next_click = world.time + CLICK_CD_MELEE
+	else ..()
```
Comment (ChangelingRain) [inline code/_onclick/click.dm]: BAD DON'T DO SAME LINE STUFF LIKE THIS

Comment (ExcessiveUseOfCobblestone) [inline code/_onclick/click.dm]: even for inheritance calls?

Comment (RemieRichards) [inline code/_onclick/click.dm]: not for anything.
the only time two statements are allowed on the same line is in macros.

Comment (xxalpha) [inline code/_onclick/click.dm]: I vomited a little.

Comment (ExcessiveUseOfCobblestone) [inline code/_onclick/click.dm]: Duly noted, I'll fix when I get to a computer

**Pull request merged**